### PR TITLE
2022 Media Chapter

### LINKED ISSUE #2884: Media 2022
# Media 2022

Media illustration

If you're interested in contributing to the Media chapter of the 2022 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- | ----------- |
| @eeeps  | @eeeps @akshay-ranganath | @nhoizey @yoavweiss | @eeeps @akshay-ranganath | @MichaelLewittes | @siakaramalegos  |

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [**0. Form the content team**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
+ [x] **May 1**: The content team has at least one author, reviewer, and analyst

### [**1. Plan content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
+ [x] **May 15** The content team has completed the chapter outline in the [draft doc][~google-doc]

### [**2. Gather data**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
+ [x] **June 1**: Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress
- **June 1 - 15**: HTTP Archive runs the June crawl

### [**3. Validate results**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
+ [x] **August 1**: Analysts have queried all metrics and saved the output to the [results sheet][~google-sheets]

### [**4. Draft content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
+ [x] **September 1**: The content team has written, reviewed, and edited the chapter in the [doc][~google-doc]

### [**5. Publication**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
+ [x] **September 15**: The completed chapter and all required [metadata](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) and [figures](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) are converted to markdown and submitted to [GitHub][~chapter-markdown]
- **September 26**: Target launch date 🚀

## Chapter resources

Refer to these 2022 Media resources throughout the content creation process:

[📄 Google Docs][~google-doc] for outlining and drafting content
[🔍 SQL files][~sql-files] for committing the queries used during analysis
[📊 Google Sheets][~google-sheets] for saving the results of queries
[📝 Markdown file][~chapter-markdown] for publishing content and managing public metadata
[💬 #web-almanac-media on Slack][~slack-channel] for team coordination

[~google-doc]: https://docs.google.com/document/d/117Roz2RlgXdDl9OiDdkrrd1ahnepFeGzIvTB5S4OY9g/edit?usp=sharing
[~sql-files]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2022/media
[~google-sheets]: https://docs.google.com/spreadsheets/d/1T5oVAVmcH3sM6R-WwH4ksr2jFtPhuLXs3-iXXoABb3E/edit?usp=sharing
[~chapter-markdown]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2022/media.md
[~slack-channel]: https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA

### PR DESCRIPTION
**Pull request by eeeps**
Closes #2884

Staging link: https://20220922t004911-dot-webalmanac.uk.r.appspot.com/en/2022/media
Home page quotes: https://20220922t004911-dot-webalmanac.uk.r.appspot.com/en/2022/?feat=media#featured-chapter

Still need to:

- [x] Add figure fallback image urls (@tunetheweb has a magic script for this?)
- [x] Add figure descriptions
- [x] Add featured stats and quotes
- [x] Couple of inline todos...
- [x] One or two more readthroughs to make sure markdown conversion went as it should.

### CODE REVIEW

**Comment by tunetheweb** [comment]
I ran my magic script and added the images. Also did some light editing.

Staged URL here: https://20220922t004911-dot-webalmanac.uk.r.appspot.com/en/2022/media (note this does not auto update but ping me if you want it updated).

**Comment by tunetheweb** [comment]
How we looking @eeps? We'll need this all complete by tomorrow (ideally today!) if we want to include in Monday's release.

**Comment by eeeps** [comment]
@tunetheweb Just added the featured quotes and gave it one more read-through; I think it's ready to merge.

**Comment by eeeps** [comment]
(we'll see if the linter agrees here in a minute or two)

**Comment by eeeps** [comment]
Did a few more tweaks and now I think it's really, actually ready to merge.

**Review by tunetheweb — APPROVED** [review]
@eeeps I made an editing pass and think this is good to merge now.

I made the following changes:
- Removed the 44% of medium bytes are images stat. This was actually 44 % of requests not bytes—it's actually larger at 63% on desktop and 64% on mobile. However that's not accurate either as the percentiles are independent and not related. Plus 3 big figures in short succession seemed a little much. So, all-in-all, I thought better to drop.
- Moved analysis to be after figures—rather than before—as per the Web Almanac style.
- More use of em-dashs for asides as per the Web Almanac style.
- Removed unecessary use of "Percentage" in captions to make them shorter.

I'll merge this later today in time for launch, but leave open for a few hours in case you want to review the edits in https://github.com/HTTPArchive/almanac.httparchive.org/pull/3143/commits/f6b7a4d12aed13e715d3adf340660a69475f622d.

I've also updated the staging link: https://20220922t004911-dot-webalmanac.uk.r.appspot.com/en/2022/media

Thanks,
Barry

**Comment by eeeps** [comment]
Thank you!!

One change I want to confirm, the intro now reads:

> We see from the Media chapter

I think that should still say "Page Weight"?

**Comment by eeeps** [comment]
Other than that the edits you made make total sense and it looks fantastic. Thank you!

File: src/content/en/2022/media.md [hunk 1]
```diff
@@ -1,22 +1,920 @@
...
-## TODO
+## Introduction
+
+Despite being hyper*text*, the web is extremely visual. In fact, images and videos are an essential part of the web user experience. They're also undergoing tremendous innovation, and the Web Almanac gives us a unique opportunity to survey both how far the visual web has come—as authors adopt new technologies such as AVIF, wide color, adaptive bitrate streaming, and lazy loading—and how far it still has to go: I'm looking at you, animated GIF.
+
+Let's dive right in.
+
+## Images
+
+Images account for a huge portion of the typical website's page weight. We see from the [Media](./media) chapter that the median website's total weight in June of 2021 was 2,019 kilobytes (on mobile), and 881 of those kilobytes were images. That's more than HTML (30 KB), CSS (72 KB), JavaScript (461 KB) and fonts (97 KB) combined.
```
Comment (tunetheweb) [inline src/content/en/2022/media.md:30]:
```suggestion
Images account for a huge portion of the typical website's page weight. We see from the [Page Weight](./page-weight) chapter that the median website's total weight in June of 2021 was 2,019 kilobytes (on mobile), and 881 of those kilobytes were images. That's more than HTML (30 KB), CSS (72 KB), JavaScript (461 KB) and fonts (97 KB) combined.
```

**Comment by tunetheweb** [comment]
> Thank you!!
> 
> One change I want to confirm, the Images section intro now reads:
> 
> > We see from the Media chapter
> 
> I think that should still say "Page Weight"?

Doh! Head non screwed on correctly. Fixed now. Will merge after CI finishes.

**Pull request merged**